Sources/SwiftCliCore/InputValidator.swift

```diff
@@ -25,7 +25,37 @@ public class InputValidator {
             throw InputError.emptyRow
         }
         return(rowStitches)
+    }
+
+    public func verifyValidRowStitchCount(prevRow: RowInfo, currentRow: RowInfo) -> Bool {
```

Comment:
Let's discuss public vs private methods and possible name chance (e.g. `isValidRowStitchCount`).